### ISSUE TITLE
Fix compatibility issues with expo SDK v42

### ISSUE THREAD

**Issue by rihards-simanovics** [rihards-simanovics/InvoiceShelf-mobile#1]
TL;DR: Need to upgrade SDK

Current SDK is outdated and incomplete, furthermore newest version of ExpoGo app required for development does not support this version of expo SDK. Additionally, although the app appears to successfully compile in development, the login appears to send corrupted or wrong request body thereby making it impossible to login, even if the credentials are correct. This was tested and compared against the crater app that's currently on the google app store and the data is being successful transmitted making me feel that something is either wrong with the source code, or the dependencies.

After attempted build multiple errors are visible in the output, without further reading the guess is that the dependencies are too old and as noted by expo-cli some APIs are likely to be deprecated.

**Comment by rihards-simanovics** [comment]
Attempted to upgrade to SDK v47 - oldest supported SDK by the latest Expo Go app. o luck, throws buffer size issue.

**Comment by rihards-simanovics** [comment]
closing in favour of redevelopment

closing in favour of #7
